gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -437,6 +492,68 @@ public static ZoneFilter notEquals(ZoneField field, String value) {
   }
 
   /**
+   * Class for filtering operation lists.
+   */
+  class OperationFilter extends ListFilter {
+
+    private static final long serialVersionUID = -3202249202748346427L;
+
+    OperationFilter(OperationField field, ComparisonOperator operator, Object value) {
+      super(field.selector(), operator, value);
+    }
+
+    /**
+     * Returns an equality filter for the given field and string value. For string fields,
```

Comment:
I think the "For string fields, ..." is slightly confusing because we use both "field" and "string" in the previous sentence to specify two different inputs.  Can we change the wording to "The string value is interpreted as..."?  (Also applies for method just below this one)